src/style/foundation/video-calling.less

```diff
@@ -175,7 +175,7 @@
       left: 50%;
       margin: 0;
       color: var(--main-color);
-      font-size: @font-size-xsmall;
+      font-size: 11px;
```

Comment:
shouldn't we use em here?

Reply:
We decided to keep a px value for some elements for now, mainly the call ui and the sidebar button